src/ape/utils.py

```diff
@@ -89,27 +91,12 @@ def get_package_version(obj: Any) -> str:
         return ""
 
 
-NOTIFY_COLORS = {
-    "WARNING": "bright_red",
-    "ERROR": "bright_red",
-    "SUCCESS": "bright_green",
-    "INFO": "blue",
-}
-
-
-def notify(type_, msg, file=None):
-    """Prepends a message with a colored tag and outputs it to the console."""
-    click.echo(
-        f"{click.style(type_, fg=NOTIFY_COLORS[type_])}: {msg}", file=file, err=type_ == "ERROR"
-    )
-
-
 class Abort(click.ClickException):
     """Wrapper around a CLI exception"""
 
     def show(self, file=None):
         """Override default ``show`` to print CLI errors in red text."""
-        notify("ERROR", self.format_message(), file=file)
+        logger.error(self.format_message())
```

Comment:
`Abort()` now logs using the global `ape` logger.
`notify()` is not used anymore! Everything uses the logger.

The `CLI` gets a decorator helper that can log for users as well as `abort()`.